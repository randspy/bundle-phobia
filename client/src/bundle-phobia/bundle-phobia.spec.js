import React from 'react';
import { shallow } from 'enzyme';
import BundlePhobia from './bundle-phobia';

describe('BundlePhobia', () => {
  it('should render correctly', () => {
    const wrapper = shallow(<BundlePhobia />);
    expect(wrapper).toMatchSnapshot();
  });
});
