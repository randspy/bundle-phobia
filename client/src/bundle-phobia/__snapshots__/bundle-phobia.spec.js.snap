// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`BundlePhobia should render correctly 1`] = `
<div>
  <Container>
    <Switch>
      <Route
        exact={true}
        path="/"
      >
        <Search />
      </Route>
      <Route
        path="/result"
      >
        <Result />
      </Route>
    </Switch>
  </Container>
</div>
`;
